loopchain/baseservice/peer_manager.py

```diff
@@ -84,6 +89,30 @@ def __init__(self, channel_name):
         self.__leader_complain_count = 0
         self.__highest_block_height = -1    # for RS heartbeat
 
+    @property
+    def peer_list(self) -> dict:
+        """
+
+        :return: { peer_id:PeerInfo }
+        """
+        # util.logger.spam(f"peer_list({self.peer_list_data.peer_info_list})")
+        return self.peer_list_data.peer_info_list
+
+    @property
+    def peer_order_list(self) -> dict:
+        """
+
+        :return: { order:peer_id }
+        """
+        return self.peer_list_data.peer_order_list
+
+    def set_peer_list(self, peer_list_data: PeerListData):
```

Comment:
If `peer_list` is intended as an interface of PeerListData, I think that the usage below is recommendable.
```
@peer_list.setter
def set_peer_list(self, peer_list_data: PeerListData):
    ....
```